salt/modules/status.py

```diff
@@ -62,10 +66,57 @@ def _number(text):
         return text
 
 
+def _get_boot_time_aix():
+    '''
+    Return the number of seconds since boot time on AIX
+
+    t=$(LC_ALL=POSIX ps -o etime= -p 1)
+    d=0 h=0
+    case $t in *-*) d=${t%%-*}; t=${t#*-};; esac
+    case $t in *:*:*) h=${t%%:*}; t=${t#*:};; esac
+    s=$((d*86400 + h*3600 + ${t%%:*}*60 + ${t#*:}))
+
+    t is 7-20:46:46
+    '''
+    boot_secs = 0
+    res = __salt__['cmd.run_all']('ps -o etime= -p 1')
```

Comment:
@s0undt3ch following the style that is already existing in the file, for example
line 432, 444, 555 etc.

The use of cmd.run with a string such as 'iostat -xzd' is rampant in the file.